ecosystem/sep-0006.md

```diff
@@ -89,8 +305,9 @@ Name | Type | Description
 `type` | string | forward
 `forward_type` | string | either {bank_account,crypto}
 `asset_code` | string | (for crypto) Specify the code of the asset you are withdrawing.
+`asset_issuer` | string | Specify the issuing account id of the asset you are withdrawing.
 `dest` | string | (for crypto) The account you want to withdraw to.
-`dest_extra` | string | (optional) If needed for other networks that might need a memo in addition to the `dest` address.
+`dest_extra` | string | (optional) If needed for other networks that might need a memo in addition to the `dest` address (most likely information obtained from the withdrawal form).
 
 On success the endpoint should return `200 OK` HTTP status code and a JSON object with the following fields:
```

Comment:
this line should be moved 4-5 lines down

Reply:
Sorry, I didn't understand.

Reply:
disregard that, not sure what I've meant :)